fbpic/lpa_utils/laser/longitudinal_laser_profiles.py

```diff
@@ -229,40 +233,71 @@ def __init__(self, z0, spectrum_file, propagation_direction=1):
             in the file. The spectral phase can be omitted, in which case it
             will be assumed to be 0.
 
+        phi2_chirp: float (in second^2), optional
+            The amount of temporal chirp, at focus (in the lab frame).
+
+        phi3_chirp: float (in second^3), optional
+            The amount of third order dispersion, at focus (in the lab frame).
```

Comment:
```suggestion
            The amount of additional third order dispersion, at focus (in the lab frame).
```
Similar comment: could you add a sentence explaining that this is *added* to the phase provided in the file?